src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14429,12 +14442,20 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                     goto Label13;
                 }
 
-                if (CapDiff > (Tolerance * Cap_Eva0)) NumIteCcap = 999;
+                if (CapDiff > (Tolerance * Cap_Eva0)) {
+                    NumIteCcap = 999;
+                    CyclingRatio = (TU_load + Pipe_Q) * C_cap_operation / Cap_Eva1;
```

Comment:
This math is confusing. (TU_load + Pipe_Q) is the load the compressor must meet. And Cap_Eva1 already includes CapFT. So shouldn't this be `(TU_load + Pipe_Q) / Cap_Eva1` ? or just `Cap_Eva0 / Cap_Eva1` ?

Reply:
`Cap_Eva0 / Cap_Eva1` probably better